src/common/buttons/IconButton.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+// the component fixes touchable feedback for react-native-elements Icon
+// the current version works only for raised=false icon.
+
```

Comment:
Despite the fact that `Icon` from react-native-elements accepts onPress prop, it doesn't provide ripple feedback and for some reason, its touchableOppacity effect isn't canceled after a tap when I use it Header. So I have to make this wrapper, its kind of `raw` because it doesn't work with `<Icon raised />` but to make it fully work I have to reimplement Icon component itself. So let it be like this for now. FYI, I stole  `TouchableItem` from react-navigation lib, I think its good enough abstraction which we can use in many places later.

Reply:
Sounds good